pkg/signup/service/signup_service_test.go

```diff
@@ -273,8 +273,8 @@ func (s *TestSignupServiceSuite) TestGetSignupFailsWithNotFoundThenOtherError()
 		s.Application.MockInformerService(inf)
 		svc := service.NewSignupService(
 			fake.MemberClusterServiceContext{
-				Client: s,
-				Svcs:   s.Application,
+				CrtClient: s,
```

Comment:
why do we need to rename the field to `CrtClient`?

Reply:
Because there is a new method called `Client` and you cannot have an exported field with the same name. At the end of the day (and this whole refactoring madness) this will all go away

Reply:
ah right, I missed that